### ISSUE TITLE
telepresence exits with "json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)"

### ISSUE THREAD

**Issue by Rangak** [telepresenceio/telepresence#731]
### What were you trying to do?

Telepresence had been working for several days. Suddenly doesn't work any more. 

 $ telepresence

### What did you expect to happen?

Was working. Uninstalled and reinstalled telepresence -- did not fix. 

### What happened instead?

(please tell us - the traceback is automatically included, see below)

### Automatically included information

Command line: `['/usr/local/bin/telepresence']`
Version: `0.90`
Python version: `3.7.0 (default, Jul  2 2018, 16:31:41) 
[Clang 9.1.0 (clang-902.0.39.2)]`
kubectl version: `Client Version: v1.11.2 // Server Version: v1.10.4`
oc version: `(error: [Errno 2] No such file or directory: 'oc': 'oc')`
OS: `Darwin My-MBP 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64`
Traceback:

```
Traceback (most recent call last):
  File "/usr/local/Cellar/telepresence/0.90/libexec/lib/python3.7/site-packages/telepresence/cli.py", line 87, in call_f
    return f(*args, **kwargs)
  File "/usr/local/Cellar/telepresence/0.90/libexec/lib/python3.7/site-packages/telepresence/main.py", line 49, in main
    session.kube_info, session.runner = analyze_args(session)
  File "/usr/local/Cellar/telepresence/0.90/libexec/lib/python3.7/site-packages/telepresence/startup.py", line 162, in analyze_args
    runner = Runner(output, kube_info.command, args.verbose)
  File "/usr/local/Cellar/telepresence/0.90/libexec/lib/python3.7/site-packages/telepresence/runner.py", line 65, in __init__
    self.cache = Cache.load(os.path.join(cache_dir, "cache.json"))
  File "/usr/local/Cellar/telepresence/0.90/libexec/lib/python3.7/site-packages/telepresence/cache.py", line 43, in load
    cache = json.load(f)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 296, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/__init__.py", line 348, in loads
    return _default_decoder.decode(s)
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 337, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/usr/local/opt/python/Frameworks/Python.framework/Versions/3.7/lib/python3.7/json/decoder.py", line 355, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

```

Logs:

```
   0.0 TEL | Telepresence 0.90 launched at Thu Aug  9 05:46:25 2018
   0.0 TEL |   /usr/local/bin/telepresence
   0.5 TEL | Context: kubernetes-admin@kubernetes, namespace: default, kubectl_command: kubectl
   0.5 TEL | [1] Launching: kubectl version --short
   0.5 TEL | [2] Launching: oc version
   0.5 TEL | [2] [Errno 2] No such file or directory: 'oc': 'oc'
   0.5 TEL | [3] Launching: uname -a
   0.5 TEL | Python 3.7.0 (default, Jul  2 2018, 16:31:41)
   0.5 TEL | [Clang 9.1.0 (clang-902.0.39.2)]
   0.5   3 | Darwin My-MBP 17.7.0 Darwin Kernel Version 17.7.0: Thu Jun 21 22:53:14 PDT 2018; root:xnu-4570.71.2~1/RELEASE_X86_64 x86_64

```

**Comment by ark3** [comment]
Sorry about the crash! The fix for this, in #716, will be in the next release. As a workaround, please delete the Telepresence cache file in ~/.cache/telepresence if it's still there.

**Comment by ark3** [comment]
Should be fixed by #716 in release 0.91.
